my_enums.rb

```diff
@@ -65,17 +143,21 @@ def my_count(item = 'CANTOR')
   end
 
   def my_map_one
+    return to_enum unless block_given?
+
     emp_arr = []
     my_each do |n|
       emp_arr << yield(n)
     end
     emp_arr
   end
 
-  def my_map_two(&block)
+  def my_map_two(proc = nil)
+    return to_enum if proc.nil?
+
     emp_arr = []
     my_each do |n|
-      emp_arr << block.call(n)
+      emp_arr << proc.call(n)
     end
     emp_arr
```

Comment:
* [ ] my_map is should be equivalent to ruby's ```map``` method
refers to [ruby doc ](https://ruby-doc.org/core-2.7.0/Enumerable.html#method-i-count)